test/reline/test_key_actor_vi.rb

```diff
@@ -1454,4 +1454,16 @@ def test_vi_kill_line_prev
     assert_cursor_max(1)
     assert_line('c')
   end
+
+  def test_vi_motion_operators
+    assert_instance_of(Reline::KeyActor::ViInsert, @config.editing_mode)
+
+    begin
+      input_keys("test = { foo: bar }\C-[BBBldt}b")
```

Comment:
without the change on line_editor.rb it throws the exception

Reply:
You can use [`assert_nothing_raised`](https://github.com/test-unit/test-unit/blob/master/lib/test/unit/assertions.rb#L683) helper for this:


```rb
  def test_vi_motion_operators
    assert_instance_of(Reline::KeyActor::ViInsert, @config.editing_mode)

    assert_nothing_raised do
      input_keys("test = { foo: bar }\C-[BBBldt}b")
    end
  end
```

Reply:
Done @st0012